src/oauth-tooling.ts

```diff
@@ -285,7 +286,10 @@ const getCredentials = (options: OAuthConfig): Promise<any>[] => {
     }
   }
 
-  return getUserData ? [getClientData, getUserData] : [getClientData];
+  return Promise.all([getUserData, getClientData])
```

Comment:
This change does not seem to return the same data any more.

Reply:
was needed to avoid a possible null access in callee